CenterForOpenScience/osf.io
# Feature/Fix Anon Logs Display for Custom Citations[ENG-589]

## Purpose

Anonymized logs related to custom citations cannot be loaded.

## Changes

Quickfix - there was a mismatch between keys in regular log list and anonymous log list, so made the keys the same. Also improved the grammar. 

## QA Notes

- Add some custom citations, edit the custom citations, and remove the citation from a project.  Verify that logs for the citation load for both a regular view and when viewed through an anonymous VOL.

### Before
<img width="547" alt="Screen Shot 2019-06-10 at 12 24 42 PM" src="https://user-images.githubusercontent.com/9755598/59214621-14581300-8b7d-11e9-8958-dbfa7c4d1f81.png">


### After
<img width="581" alt="Screen Shot 2019-06-10 at 12 24 35 PM" src="https://user-images.githubusercontent.com/9755598/59214618-128e4f80-8b7d-11e9-81fa-6a8396a8a284.png">


## Documentation

None needed

## Side Effects



## Ticket

https://openscience.atlassian.net/browse/ENG-589
